tests/resolve_select_expression.rs

```diff
@@ -240,6 +240,7 @@ baz-unknown =
 }
 
 #[test]
+#[ignore]
```

Comment:
Why do you need to ignore this test?

Note that [message references cannot be used as selectors](https://github.com/projectfluent/fluent/blob/69ecabcc06cf6f5b3654c439a9defc8575744313/spec/fluent.ebnf#L67-L71) in Syntax 0.5. The `use-foo` message should be updated.